unofficial_flocker_tools/config.py

```diff
@@ -143,6 +144,36 @@ def main(reactor, args):
 
     yield gatherResults(deferreds)
 
+    if c.config["os"] == "ubuntu":
+        if len(sys.argv) > 2:
+            if sys.argv[2] == "--swarm":
```

Comment:
Probably worth introducing `twisted.usage.Options` for proper args parsing pretty soon, if not immediately. See https://github.com/ClusterHQ/unofficial-flocker-tools/blob/master/unofficial_flocker_tools/flocker_volumes.py#L83 and friends for sample `optArgs` usage.